packages/vincent-tools/vincent-tool-uniswap-swap/buffer.shim.js

```diff
@@ -0,0 +1,2 @@
+import { Buffer } from 'buffer';
```

Comment:
As above, I think this was replaced with `esbuild-plugin-polyfill-node`

We should export the esbuild.config.js from the `vincent-tools-sdk` package so that all of our tools/policies bundle using the same code and we don't need to update all of them independently -- consumers of the tool SDK can also import these into their projects rather than have to write it themselves.